protokt-runtime/build.gradle.kts

```diff
@@ -15,7 +15,7 @@
 
 enablePublishing()
 compatibleWithAndroid()
-trackKotlinApiCompatibility()
+trackKotlinApiCompatibility(false)
```

Comment:
why do we need this if we are adding to the API?

Reply:
It complains if you change the public API at all.

```
Caused by: java.lang.IllegalStateException: API check failed for project protokt-runtime.
--- /root/code/protokt-runtime/build/reference-api/protokt-runtime.api
+++ /root/code/protokt-runtime/build/api/protokt-runtime.api
@@ -13,6 +13,10 @@
 	public final fun empty ()Lcom/toasttab/protokt/rt/Bytes;
 }
 
+public final class com/toasttab/protokt/rt/BytesKt {
+	public static final fun toBytesSlice (Lcom/toasttab/protokt/rt/Bytes;)Lcom/toasttab/protokt/rt/BytesSlice;
+}
+
 public final class com/toasttab/protokt/rt/BytesSlice {
 	public static final field Companion Lcom/toasttab/protokt/rt/BytesSlice$Companion;
 	public fun <init> ([B)V
@@ -29,6 +33,10 @@
 	public final fun empty ()Lcom/toasttab/protokt/rt/BytesSlice;
 }
 
+public final class com/toasttab/protokt/rt/BytesSliceKt {
+	public static final fun toBytes (Lcom/toasttab/protokt/rt/BytesSlice;)Lcom/toasttab/protokt/rt/Bytes;
+}
+
 public final class com/toasttab/protokt/rt/Fixed32 {
 	public static final synthetic fun box-impl (I)Lcom/toasttab/protokt/rt/Fixed32;
 	public static fun constructor-impl (I)I

 You can run :protokt-runtime:apiDump task to overwrite API declarations
```